### PR TITLE
Update Documentation + Deployment Test Delay

### PR DESCRIPTION
**Pull request by AlexBMes**
* Update documentation with more details on secure swarm node configuration.
* For deployment test bug: add 5 minute delay before deploying App Engine in the MCP deployment test, to ensure default service account is created before trying to give it permissions.

### CODE REVIEW

**Review by gbmeuk — APPROVED** [review]
LGTM but will before merging, get @danperezsan final approval

**Review by danperezsan — APPROVED** [review]

**Pull request merged**
